data-prepper-plugins/rds-source/src/test/java/org/opensearch/dataprepper/plugins/source/rds/export/ExportTaskManagerTest.java

```diff
@@ -41,45 +45,29 @@ void setUp() {
         exportTaskManager = createObjectUnderTest();
     }
 
-    @Test
-    void test_start_export_task_without_specifying_tables() {
-        final String snapshotArn = "arn:aws:rds:us-east-1:123456789012:snapshot:snapshot-0b5ae174";
-        final String iamRoleArn = "arn:aws:iam:us-east-1:123456789012:role:my-role";
-        final String bucket = "bucket";
-        final String prefix = "prefix";
+    @ParameterizedTest
+    @MethodSource("provideStartExportTaskTestParameters")
```

Comment:
I recommend using `@ArgumentsSource` which takes in a class name. You'd have to make the method an `ArgumentsProvider` class. But, it creates a tighter link since it has more compile-time validations. But, I'm fine keeping this.

Thanks for using a parameterized test!

Reply:
Thanks for the suggestion! Started using the @ArgumentSource in the new ExportStatusTest.